smart-assembly-scaffold/packages/contracts/.envsample

```diff
@@ -1,19 +1,19 @@
 # This .env file is for demonstration purposes only.
 #
 # This should usually be excluded via .gitignore and the env vars attached to
-# your deployment environment, but we're including this here for ease of local
+# your deployment enviroment, but we're including this here for ease of local
 # development. Please do not commit changes to this file!
 #
 # Enable debug logs for MUD CLI
 DEBUG=mud:*
 #
 # Anvil default private key:
-#Deployer
+#Deployer 
 PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 WORLD_ADDRESS=0x8a791620dd6260079bf849dc5567adc3f2fdc318
 
 #Player interacting with the SSU
-TEST_PLAYER_PRIVATE_KEY=0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
+TEST_PLAYER_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
See comment below in `SetPrivateKey.sh`. I think there is value in setting the default for the interacting player to the second Anvil test account.